packages/optimizer/spec/transformers/valid/ReorderHeadTransformer/supports_esm/expected_output.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <link as="script" crossorigin="anonymous" href="https://cdn.ampproject.org/v0.mjs" rel="modulepreload">
+  <link rel="preload" href="https://cdn.ampproject.org/v0.css" as="style">
+  <link rel="stylesheet" href="https://cdn.ampproject.org/v0.css">
+  <script async nomodule src="https://cdn.ampproject.org/v0.js"></script>
+  <script async src="https://cdn.ampproject.org/v0.mjs" type="module" crossorigin="anonymous"></script>
+  <script async nomodule src="https://cdn.ampproject.org/v0/amp-experiment-0.1.js" custom-element="amp-experiment"></script>
+  <script async custom-element="amp-experiment" src="https://cdn.ampproject.org/v0/amp-experiment-0.1.mjs" type="module" crossorigin="anonymous"></script>
+  <script async nomodule src="https://cdn.ampproject.org/v0/amp-mustache-0.2.js" custom-template="amp-mustache"></script>
+  <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.2.mjs" type="module" crossorigin="anonymous"></script>
```

Comment:
Can the order be reversed, with the `nomodule` one coming after?

Reply:
done